frontend/javascripts/components/link_button.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+import { Button } from "antd";
+import * as React from "react";
+
+export default function LinkButton(props: Object) {
```

Comment:
```suggestion
// This component should be used instead of <a href="#" onClick={} />.
// One advantage of this component is that clicking the link button won't scroll
// up (which would happen with <a ... /> if not handled otherwise)
export default function LinkButton(props: Object) {
```